# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../stella/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- stella-2.6.1/src/unix/SerialPortUNIX.cxx.vanilla	2008-05-29 10:33:48.000000000 +0200
+++ stella-2.6.1/src/unix/SerialPortUNIX.cxx	2008-05-29 10:33:57.000000000 +0200
@@ -18,6 +18,7 @@
 
 #include <sys/types.h>
 #include <sys/stat.h>
+#include <string.h>
 #include <fcntl.h>
 #include <unistd.h>
 #include <sys/termios.h>
--- stella-2.6.1/src/emucore/Cart.cxx.vanilla	2008-05-29 10:34:44.000000000 +0200
+++ stella-2.6.1/src/emucore/Cart.cxx	2008-05-29 10:34:50.000000000 +0200
@@ -16,6 +16,7 @@
 // $Id: Cart.cxx,v 1.42 2008/05/16 23:56:21 stephena Exp $
 //============================================================================
 
+#include <cstring>
 #include <cassert>
 #include <sstream>
 
--- stella-2.6.1/src/emucore/TrackBall.cxx.vanilla	2008-05-29 10:35:13.000000000 +0200
+++ stella-2.6.1/src/emucore/TrackBall.cxx	2008-05-29 10:35:44.000000000 +0200
@@ -16,6 +16,8 @@
 // $Id: TrackBall.cxx,v 1.2 2008/05/19 03:15:12 stephena Exp $
 //============================================================================
 
+#include <cstdlib>
+
 #include "Event.hxx"
 #include "System.hxx"
 #include "TIA.hxx"
--- stella-2.6.1/src/emucore/CartAR.cxx.vanilla	2008-05-29 10:34:27.000000000 +0200
+++ stella-2.6.1/src/emucore/CartAR.cxx	2008-05-29 10:34:35.000000000 +0200
@@ -16,6 +16,7 @@
 // $Id: CartAR.cxx,v 1.21 2008/03/28 23:29:13 stephena Exp $
 //============================================================================
 
+#include <cstring>
 #include <cassert>
 
 #include "M6502Hi.hxx"
